GenderPayGap.WebUI/wwwroot/styles/components/_guidance.scss

```diff
@@ -60,3 +60,22 @@
         }
     }
 }
+
+.print-link {
+    background: url(../../img/icon-print.png) no-repeat 0 50%;
+    padding: 0.5em 0 0.5em 28px
```

Comment:
Having a mix of `em` and `px` feels wrong to me. Also, the `28px` value for the left-padding should depend on the size of the icon, and therefore the font size. can you convert it to em?